nuxt-frontend/components/backend-development/PartnerWithUsSection.vue

```diff
@@ -0,0 +1,216 @@
+              :class="{ '!flex': isAnimated }"
+            >
+              <span
+                class="header-text"
+                :class="{
+                  '!text-white': isAnimated,
+                }"
+                >Why should you</span
+              >
+              <span
+                class="block ml-2 transition-all duration-1000"
+                :class="[
+                  isAnimated ? '!text-white !header-text !ml-0' : 'text1',
+                ]"
+                >partner
+              </span>
+              <div
+                class="flex ml-0 xl:ml-32 gap-6 transition-all duration-1000"
+                :class="{ '!ml-0 block': isAnimated }"
+              >
+                <span
+                  class="my-auto -rotate-90 header-text transition-all duration-1000"
+                  :class="{
+                    '!text-white rotate': isAnimated,
+                  }"
+                  >with</span
+                ><span
+                  class="transition-all duration-1000"
+                  :class="[isAnimated ? 'text-white !header-text' : 'text1']"
+                  >Canopas?</span
+                >
+              </div>
+              <nuxt-link
+                class="gradient-btn primary-btn ml-0 mt-10 animate-fadeIn"
+                :class="[!isAnimated ? 'hidden' : '']"
+                to="/contact"
+                @click.native="
+                  $mixpanel.track('tap_backend_partner_with_us_cta')
+                "
+              >
+                <span class="sub-h3-semibold lg:sub-h1-semibold"
+                  >Let’s Work Together</span
+                ></nuxt-link
+              >
+            </div>
+            <nuxt-link
+              class="gradient-btn primary-btn mt-[5.69rem]"
+              :class="{ '!hidden': isAnimated }"
+              to="/contact"
+              @click.native="$mixpanel.track('tap_backend_partner_with_us_cta')"
+            >
+              <span class="sub-h3-semibold lg:sub-h1-semibold"
+                >Let’s Work Together</span
+              ></nuxt-link
+            >
+          </div>
+          <div
+            class="flex flex-col absolute w-[50%] left-1/2 -translate-y-1/2 top-1/2 overflow-hidden"
+          >
+            <div
+              v-for="(item, index) in items"
+              :key="item"
+              class="flex flex-col mb-10 item"
+              :class="[
+                isVisible
+                  ? `item-${index} transform-all duration-1000 opacity-100`
+                  : 'translate-x-full',
+              ]"
+            >
+              <div class="flex gap-[1.69rem]">
+                <img
+                  :src="item.image"
+                  class="w-9 h-9 xl:w-14 xl:h-14 object-contain"
+                  :alt="item.title"
+                />
+                <div class="flex flex-col gap-4">
+                  <span
+                    class="text-xl leading-8 font-semibold xl:mobile-header-2-semibold"
```

Comment:
here don't use this extra class use class from available classes only which will fit as home page lg design